model/view-active-patients.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/**
+ * Registers view with Sequelize
+ * @param {Sequelize} sequelize - database instance
+ * @param {String} salt - salt value to use when hashing password
+ * @returns {Null} nothing
+ */
+function register (sequelize) {
+    /**
+     * patient that has not been deactivated
+     * @typedef {Object} ActivePatient
+     */
+    sequelize.query(
+        `
+        CREATE OR REPLACE VIEW active_patients AS
```

Comment:
This looks like a good name to me. Active patients are the ones which are currently active in the system